client/components/jetpack/daily-backup-status/status-card/backup-failed.jsx

```diff
@@ -26,38 +27,65 @@ const BackupFailed = ( { backup } ) => {
 	const displayDate = backupDate.format( 'L' );
 	const displayTime = backupDate.format( 'LT' );
 
+	const mayBeBlockedByHost = getBackupErrorCode( backup ) === 'NOT_ACCESSIBLE';
+
 	return (
 		<>
 			<div className="status-card__message-head">
 				<img src={ cloudErrorIcon } alt="" role="presentation" />
 				<div className="status-card__message-error">{ translate( 'Backup failed' ) }</div>
 			</div>
-			<div className="status-card__title">{ getDisplayDate( backup.activityTs, false ) }</div>
+			<div className="status-card__title">
+				{ mayBeBlockedByHost
+					? translate( `We'are having trouble backing up your site` )
```

Comment:
Just noticed a small typo here in `We'are`.  I think it should read `We're` or `We are`.

Reply:
Oh, thank you for catching that.